<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureUi</class>
 <widget class="QWidget" name="ConfigureUi">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>280</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="general_groupBox">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_change_language_info">
          <property name="text">
           <string>Note: Changing language will apply your configuration.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="language_label">
            <property name="text">
             <string>Interface language:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="language_combobox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="theme_label">
            <property name="text">
             <string>Theme:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="theme_combobox"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="game_list_groupBox">
     <property name="title">
      <string>Game List</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="icon_size_label">
            <property name="text">
             <string>Icon Size:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="icon_size_combobox">
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Small (24x24)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Large (48x48)</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="row_1_text_label">
            <property name="text">
             <string>Row 1 Text:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="row_1_text_combobox">
            <item>
             <property name="text">
              <string>File Name</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Full Path</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Title Name (short)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Title ID</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Title Name (long)</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="row_2_text_label">
            <property name="text">
             <string>Row 2 Text:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="row_2_text_combobox">
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>File Name</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Full Path</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Title Name (short)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Title ID</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Title Name (long)</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="toggle_hide_no_icon">
          <property name="text">
           <string>Hide Titles without Icon</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="toggle_single_line_mode">
          <property name="text">
           <string>Single Line Mode</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
